src/ngraph/frontend/onnx_import/op/expand.cpp

```diff
@@ -42,18 +35,11 @@ namespace ngraph
                     const std::shared_ptr<ngraph::Node> data{node.get_ng_inputs().at(0)};
                     const std::shared_ptr<ngraph::Node> shape{node.get_ng_inputs().at(1)};
 
-                    NGRAPH_CHECK(shape->is_constant(),
-                                 "Ngraph does not support dynamic braodcasting for Expand op.");
+                    const auto const_filled_with_ones = std::make_shared<default_opset::Broadcast>(
+                        default_opset::Constant::create(data->get_element_type(), {}, {1}), shape);
 
-                    std::vector<std::size_t> shape_vector =
-                        ngraph::as_type_ptr<default_opset::Constant>(shape)
-                            ->get_vector<std::size_t>();
-
-                    const ngraph::Shape shape_shape{shape_vector};
-                    return {std::make_shared<default_opset::Broadcast>(
-                        data,
-                        default_opset::Constant::create(
-                            element::i64, Shape{shape_shape.size()}, shape_shape))};
+                    return {
+                        std::make_shared<default_opset::Multiply>(data, const_filled_with_ones)};
```

Comment:
Why not just using `v1::Broadcast` ??? 
https://github.com/NervanaSystems/ngraph/blob/1f2ae36f44d48f9ea341bd5156726896ef920353/src/ngraph/op/broadcast.hpp#L150-L153